utils/types.go

```diff
@@ -156,10 +131,15 @@ func GetKongClient(opt KongClientConfig) (*kong.Client, error) {
 	defaultTransport.TLSClientConfig = &tlsConfig
 	c.Transport = defaultTransport
 	if len(opt.Headers) > 0 {
-		c.Transport = &HeaderRoundTripper{
-			headers: opt.Headers,
-			rt:      defaultTransport,
+
+		headers := http.Header{}
+		for _, keyValue := range opt.Headers {
+			split := strings.SplitN(keyValue, ":", 2)
+			if len(split) >= 2 {
+				headers[split[0]] = []string{split[1]}
```

Comment:
`http.Header` has an `Add()` function in its public interface.
Its behavior is slightly different than what is here now: `Add()` canonicalizes the header key format.

Should we use `Add()` instead of a map assignment here? If not, why not?